### PR TITLE
Add first README

### LINKED ISSUE #13: Add README
Should explain the app, reqs such as formatter, and names of branches and links to curriculum.

### PR DESCRIPTION
**Pull request by mark-yacoub**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## About the Tree
+As this is a Coptic learning app, some common terms have been changed to their Coptic meanings:
+The **master** branch has been renamed to **neeb** (Ⲛⲏⲃ).
+Git **origin** renamed to **moumi** (Ⲙⲟⲩⲙⲓ).
+So, when you pull master it will be `git pull neeb`
+and when you push to origin it will be `git push moumi`
+
+## About the formatter & packages
+We use `ESLint` and `Prettier`, download the extensions on your editor.
+We use `yarn` rathern than `npm install`
```
Comment (ysawires) [inline README]: Minor comment, fix "rather"

**Review by ysawires — CHANGES_REQUESTED** [review]

**Review by markusamuel — APPROVED** [review]

File: README (hunk 1 above)
Comment (mark-yacoub) [reply README:23]: Thanks!

Comment (mark-yacoub) [reply README]: Thanks!

**Comment by mark-yacoub** [comment]
fixed!

**Pull request merged**